### DISCUSSION TITLE
[Docs]在鱼骨图布局文档中，没有找到属性：isLayoutInvisibleNodes 和 preLayout的来源。

### DISCUSSION THREAD

**Discussion by liaodalin19903** [antvis/G6#6678] [Q&A]
在阅读[鱼骨图布局文档](https://g6.antv.antgroup.com/api/layouts/fishbone#type)的时候：
有看到2个布局的设置项目在代码中没有找到内容：
```
isLayoutInvisibleNodes
preLayout
```

不管从[FishboneLayoutOptions](https://github.com/antvis/G6/blob/v5/packages/g6/src/layouts/fishbone.ts#L9)
还是从它的继承的BaseLayoutOptions 都没有找到有相关的属性。

**Comment by petercat-assistant** [comment]
在鱼骨图布局文档中提到的属性 `isLayoutInvisibleNodes` 和 `preLayout` 似乎在代码中没有找到相关的定义。这可能是文档与代码实现之间的不同步或遗漏。

为了更好地理解这些属性的用途和来源，我们可以采取以下步骤：

1. **检查代码库**：查看 `FishboneLayoutOptions` 和 `BaseLayoutOptions` 的定义，确认这些属性是否在代码中被定义或使用。
2. **查看更新日志或提交记录**：有时候属性可能在某个版本中被移除或重命名，查看更新日志或提交记录可以帮助我们了解这些属性的历史变化。
3. **联系维护者或提交问题**：如果确认这些属性确实缺失，可以在 GitHub 上提交一个问题，询问这些属性的具体用途和实现计划。

如果需要，我可以帮助你在代码库中搜索这些属性，看看是否有遗漏或其他相关信息。请告诉我你的决定。
